README.md

```diff
@@ -1,52 +1,113 @@
 # charuco_calibration
+
 Camera calibration via charuco board ROS package.
-> If you want to calibrate the camera on the embedded computer (like the Raspberry Pi), you should install this package on your laptop and use remote calibration launch file. 
+> If you want to calibrate the camera on the embedded computer (like the Raspberry Pi), you should install this package on your laptop and use remote calibration launch file.
+
+Currently the package provides the following nodes:
+
+* `charuco_calibration_node` - an interactive ChArUco calibration node that uses an image topic to get calibration images;
```

Comment:
Why not to merge it to one node?

Reply:
I'd prefer the nodes to be simplier, with less code paths, less code overall. Besides, right now the two nodes have significantly different behaviour: one of them expects an image topic and user input, the other just looks at images in a folder and creates a calibration file.